plugins/trace_api_plugin/request_handler.cpp

```diff
@@ -42,89 +43,120 @@ namespace {
          yield();
 
          const auto& a = actions.at(index);
-         auto action_variant = fc::mutable_variant_object()
-               ("global_sequence", a.global_sequence)
+         auto common_mvo = fc::mutable_variant_object();
+         common_mvo("global_sequence", a.global_sequence)
                ("receiver", a.receiver.to_string())
                ("account", a.account.to_string())
                ("action", a.action.to_string())
                ("authorization", process_authorizations(a.authorization, yield))
                ("data", fc::to_hex(a.data.data(), a.data.size()));
-
-         auto params = data_handler(a, yield);
-         if (!params.is_null()) {
-            action_variant("params", params);
+         auto action_variant = fc::mutable_variant_object();
+         if constexpr(std::is_same_v<ActionTrace, action_trace_v0>){
+            action_variant(common_mvo);
```

Comment:
`common_mvo` could be `std::move` into `action_variant`. However, not sure why you are not just appending to `action_variant` instead of creating a temp that you just create `action_variant` from.

Reply:
I forgot to use std::move, this make sense.